tool/tbot/main.go

```diff
@@ -144,6 +155,10 @@ func onWatch(botConfig *config.BotConfig) error {
 }
 
 func onStart(botConfig *config.BotConfig) error {
+	if botConfig.AuthServer == "" {
+		return trace.BadParameter("An auth server must be set via --auth-server or configuration")
```

Comment:
Do we support tunneling through proxy? If so, I recommend we change the wording to mention that it can be an auth or proxy server.

Reply:
Good point, will reword.